platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/pipeline/VirtualPipeline.java

```diff
@@ -473,7 +473,6 @@ private long currentTotalSize() {
         for (PipelineListNode<VirtualRoot> node = copies.getFirst(); node != null; node = node.getNext()) {
             final VirtualRoot copy = node.getValue();
             if (!copy.isImmutable()) {
-                assert node.getNext() == null;
```

Comment:
This is a thread race. Right after `copy.isImmutable()` check, the copy may become mutable, and a new copy may become its "next" here. I noticed this issue when one of VirtualPipelineTests cases failed a couple times in a row.